node/node.go

```diff
@@ -1061,6 +1060,7 @@ func (app *App) initServices(ctx context.Context) error {
 		activation.WithPostValidityDelay(app.Config.PostValidDelay),
 		activation.WithPostStates(postStates),
 		activation.WithPoets(poetClients...),
+		activation.BuilderAtxVersions(app.Config.AtxVersions),
```

Comment:
Does `AtxVersions` config field have any validation currently? It would make sense to check it

Reply:
It's validated when parsed from the config: 

https://github.com/spacemeshos/go-spacemesh/blob/362aab0fe58035c746456d5e0d4714661e4e49c1/activation/handler.go#L53-L66

https://github.com/spacemeshos/go-spacemesh/blob/f9119e12f9abe605d05cbe523c13cdcff2d39bee/node/mapstructureutil/atxversions.go#L11-L30